osbs/build/build_requestv2.py

```diff
@@ -24,7 +30,186 @@
 logger = logging.getLogger(__name__)
 
 
-class BuildRequestV2(object):
+@six.add_metaclass(abc.ABCMeta)
+class BaseBuildRequest(object):
+    """
+    Abstract class for all build requests
```

Comment:
Should the docstring describe `self.user_params` and the `render` method?

Reply:
added